### DISCUSSION TITLE
Hi, trying to understand Projections.

### DISCUSSION THREAD

**Discussion by laredoza** [JasperFx/marten#3044] [Q&A]
I get the following error

<img width="969" alt="image" src="https://github.com/JasperFx/marten/assets/5489412/7bb22713-7ee5-42fa-8cbe-3b3c8ed3187b">

`
var a = session.Events.AggregateStream<LeadCountProjection>(id);
`

Startup code:

<img width="697" alt="image" src="https://github.com/JasperFx/marten/assets/5489412/750fe3e7-daf6-4b2d-8c43-664cfbf44763">

Projection:

<img width="691" alt="image" src="https://github.com/JasperFx/marten/assets/5489412/f3f8be2b-2faa-494a-a3f2-6185503be483">

Class:

<img width="496" alt="image" src="https://github.com/JasperFx/marten/assets/5489412/8902217b-c3b3-4276-8ffe-a13cefa9d695">

I'd appreciate any help getting this to work.

**Comment by gfoidl** [comment]
You registered the projection with livecylce `Live`, but it should be `Inline` or `Async` here.

For live-aggregations see [docs](https://martendb.io/events/projections/live-aggregates.html) and how to have the `Apply`-methods.
